files/build_templates/docker_image_ctl.j2

```diff
@@ -14,15 +14,15 @@ link_namespace() {
     mkdir -p /var/run/netns
     PID="$(docker inspect -f {{"'{{.State.Pid}}'"}} "{{docker_container_name}}$DEV")"
 
-    if `ip netns | grep --quiet -w "{{docker_container_name}}$DEV"`; then # namespace exists
-        if [ $(readlink -f /var/run/netns/$NET_NS$DEV) = $(readlink -f /proc/$PID/ns/net) ]; then # namespace is correctly linked
+    if `ip netns | grep --quiet -w "$NET_NS"`; then # namespace exists
```

Comment:
>ip netns | grep --quiet -w "$NET_NS" [](start = 8, length = 36)

suggestion use `ip netns pids` return value to determine exists

Reply:
Fixed as per comment

Reply:
The comment is useful
` # namespace exists`

---
In reply to: [387993327](https://github.com/Azure/sonic-buildimage/pull/3888#discussion_r387993327) [](ancestors = 387993327)

Reply:
Fixed as per comment.